### ISSUE TITLE
Unable to save metadata file (only in combination with wrap plugin)

### ISSUE THREAD

**Issue by ArtMel1998** [selfthinker/dokuwiki_plugin_wrap#232]
Hi there,

I have DokuWiKi installed in a Docker container on a Synology NAS (1819+). On a couple of pages I have used the wrap plugin with colums
```
<WRAP group>
<WRAP half column>

<WRAP left round box 90%>
  Information
</WRAP>

<WRAP half column>

<WRAP left round box 90%>
  Information
</WRAP>

</WRAP>
</WRAP>
```
Now when saving this page it generates an error on top unable to save metadata file. Hint: Disk Full (checked that, plenty of space), File permissions (I am the administrator of this NAS with full control), safe_mode setting (I am not in safe mode), 
The funny thing is: The page gets saved, it only seems to have issues with saving the metadata.

This error only appears on the pages whit the wrap plugin, on other pages I don't see this.

Some detailed information:
Docker version: 20.10.2-1239
DoKuWiki version: 2020-07-29
Template: Bootstrap v2021-03-11
Wrap plugin verion 2018-04-22 (did a reinstall which did not update the version to a newer one)
User: administrator with admin rights (also on the folders that contain the DokuWiki installation)

**Comment by ArtMel1998** [comment]
Today I analyzed a bit further and was able to get rid of the warnings. Don't know why though. Here's what I did:

- made a copy of the folder with the name wachtwoorden
- renamed the original folder to _wachtwoorden. This gave a odd result where the pages still get loaded, this should not be the case for within DokuWiki I still refer to the folder wachtwoorden and not to the folder _wachtwoorden. This can only mean that DokuWiki still keeps track although it said it can't.
- deleted the folder _wachtwoorden and renamed in the sidebar the pagenames to something else. Now DokuWiki tells me it can't find any pages yet and that I should create them. Oke, this is what I expected :).
- created a new folder with the name passwords (this is what wachtwooden is in Dutch)
- changed the sidebar and made new pages from a template I use.

The result now is that I have the pages without the error and with the template information. The next I did is to copy the old information back to the new pages and save the pages. At that time DokuWiki tells me a newer version exists and ask me if I'm sure that I want to overwrite the page. When I click yes I have the original page back without errors.

So for me it is solved, only now I have no idea waht causes the warnings all together for it seam not to be wrapper plugin related.
